fe/lib/components/LocationSelectMap/index.ts

```diff
@@ -0,0 +1 @@
+export { LocationSelectMap } from './LocationSelectMap';
```

Comment:
I think you'll want this export in the root `index.ts`: https://github.com/seek-oss/wingman/blob/master/fe/lib/index.ts